src/views/registration/attendees/fields/text.php

```diff
@@ -0,0 +1,37 @@
+<?php
+/**
+ * This template renders a Single Ticket content
+ * composed by Title and Description currently
+ *
+ * @version TBD
+ *
+ */
+$attendee_id   = $key;
+$required      = isset( $field->required ) && 'on' === $field->required ? true : false;
+$option_id     = "tribe-tickets-meta_{$field->slug}" . ( $attendee_id ? '_' . $attendee_id : '' );
+$field         = (array) $field;
+$multiline     = isset( $field['extra'] ) && isset( $field['extra']['multiline'] ) ? $field['extra']['multiline'] : '';
+$is_restricted = false;
```

Comment:
You set this, but never change it - is it necessary?

Reply:
At this point I think it's not. We inherited these from ET+ templates, and at this point the fields for the attendee registration page cannot be disabled.